scripts/kubestellar

```diff
@@ -86,7 +86,55 @@ then
     exit 1
 fi
 
+# current ws at start does not matter, is root:compute on return
+function ensure_compute_status_updateable() {
+    kubectl ws root:compute &> /dev/null
+    kubectl apply -f - &> /dev/null <<EOF
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: compute:apiexport:kubernetes:maximal-permission-policy-extended
+rules:
+- apiGroups:
+  - ""
+  - apps
+  - networking.k8s.io
+  resources:
+  - services/status
+  - pods/status
+  - ingresses/status
+  - deployments/status
+  verbs:
+  - update
+  - patch
+- apiGroups:
+  - apps
+  resources:
+  - deployments/scale
+  verbs:
+  - get
+  - update
+  - patch
+EOF
+    kubectl apply -f - &> /dev/null <<EOF
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: compute:authenticated:apiexport:kubernetes:maximal-permission-policy-extended
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: compute:apiexport:kubernetes:maximal-permission-policy-extended
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: User
+  name: kcp-admin
```

Comment:
Could you please correct the subject to the following one? 
```
subjects:
- apiGroup: rbac.authorization.k8s.io
  kind: Group
  name: apis.kcp.io:binding:system:authenticated
```

Reply:
done